File: AppGenerateFiles/dbo/Model/Transaction_Facturas.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Transaction_Facturas : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? numero_factura { get; set; }
       public Double? abono_de_cuota { get; set; }
       public Double? mora { get; set; }
       public Double? interes { get; set; }
       public Double? total { get; set; }
       public int? numero_contrato { get; set; }
       public DateTime? fecha { get; set; }
       public DateTime? fecha_pago { get; set; }
       public Double? inte { get; set; }
       public Double? mor { get; set; }
       public int? dm { get; set; }
       public string? es { get; set; }
       public Double? tot { get; set; }
       public string? an { get; set; }
       public Double? pago_contado { get; set; }
       public Double? saldo_monto { get; set; }
       public Double? ABONO { get; set; }
       public Double? descuento { get; set; }
       public DateTime? fecha_mora { get; set; }
       public DateTime? fecha_interes { get; set; }
       public Double? taza_cambio { get; set; }
       public Double? interes_actual { get; set; }
       public int? Id_User_OLD { get; set; }
       public DateTime? fecha_grabado { get; set; }
       public DateTime? mes_pagado { get; set; }
       public DateTime? ultima_visita { get; set; }
       public Double? dmpagadas { get; set; }
       public string? tipo { get; set; }
       public Double? morac { get; set; }
       public Double? interesc { get; set; }
       public Double? abonoc { get; set; }
       public Double? totalc { get; set; }
       public Double? parciales { get; set; }
       public Double? moraparcial { get; set; }
       public Double? interesparcial { get; set; }
       public string? motivo_anulacion { get; set; }
       public Double? reestructuraciond { get; set; }
       public Double? reestructuracionc { get; set; }
       public int? numero_reestructuracion { get; set; }
       public DateTime? fecha_cancelacion { get; set; }
       public Double? docnoentregadod { get; set; }
       public Double? docnoentregadoc { get; set; }
       public int? Id_User { get; set; }
       [ManyToOne(TableName = "Security_Users", KeyColumn = "Id_User", ForeignKeyColumn = "Id_User")]
       public Security_Users? Security_Users { get; set; }
   }
}

File: AppGenerateFiles/dbo/FrontModel/Transaction_Facturas.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { Security_Users }  from './Security_Users.js'
class Transaction_Facturas extends EntityClass {
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ numero_factura;
   /**@type {Number}*/ abono_de_cuota;
   /**@type {Number}*/ mora;
   /**@type {Number}*/ interes;
   /**@type {Number}*/ total;
   /**@type {Number}*/ numero_contrato;
   /**@type {Date}*/ fecha;
   /**@type {Date}*/ fecha_pago;
   /**@type {Number}*/ inte;
   /**@type {Number}*/ mor;
   /**@type {Number}*/ dm;
   /**@type {String}*/ es;
   /**@type {Number}*/ tot;
   /**@type {String}*/ an;
   /**@type {Number}*/ pago_contado;
   /**@type {Number}*/ saldo_monto;
   /**@type {Number}*/ ABONO;
   /**@type {Number}*/ descuento;
   /**@type {Date}*/ fecha_mora;
   /**@type {Date}*/ fecha_interes;
   /**@type {Number}*/ taza_cambio;
   /**@type {Number}*/ interes_actual;
   /**@type {Number}*/ Id_User_OLD;
   /**@type {Date}*/ fecha_grabado;
   /**@type {Date}*/ mes_pagado;
   /**@type {Date}*/ ultima_visita;
   /**@type {Number}*/ dmpagadas;
   /**@type {String}*/ tipo;
   /**@type {Number}*/ morac;
   /**@type {Number}*/ interesc;
   /**@type {Number}*/ abonoc;
   /**@type {Number}*/ totalc;
   /**@type {Number}*/ parciales;
   /**@type {Number}*/ moraparcial;
   /**@type {Number}*/ interesparcial;
   /**@type {String}*/ motivo_anulacion;
   /**@type {Number}*/ reestructuraciond;
   /**@type {Number}*/ reestructuracionc;
   /**@type {Number}*/ numero_reestructuracion;
   /**@type {Date}*/ fecha_cancelacion;
   /**@type {Number}*/ docnoentregadod;
   /**@type {Number}*/ docnoentregadoc;
   /**@type {Security_Users} ManyToOne*/ Security_Users;
}
export { Transaction_Facturas }
